src/Form/Control/Lookup.php

```diff
@@ -378,22 +378,22 @@ protected function initDropdown($chain): void
     protected function renderView(): void
     {
         if ($this->multiple) {
-            $this->template->set('multiple', 'multiple');
+            $this->template->set('multiple', 'multiple="multiple"');
         }
 
         if ($this->disabled) {
             $this->settings['showOnFocus'] = false;
             $this->settings['allowTab'] = false;
 
-            $this->template->set('disabled', 'disabled');
+            $this->template->set('disabled', 'disabled="disabled"');
```

Comment:
must be set as html:

![image](https://user-images.githubusercontent.com/2228672/187190497-69bfaae4-e83c-4659-9b1e-1e916daacba7.png)

must investigate, `"` does need to be escaped only in attributes

Reply:
fixed, mixed tag use in Lookup template